### PR TITLE
52 county level data

### PR DESCRIPTION
**Pull request by ghop02**

### CODE REVIEW

File: libs/datasets/data_source.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    POPULATION_FIELD_MAP = None
+
+    # Name of dataset source
+    SOURCE_NAME = None
+
+    def __init__(self, data: pd.DataFrame):
+        self.data = data
+
+    @classmethod
+    def build_from_local_github(cls) -> "cls":
```
Comment (crwilcox) [inline libs/datasets/data_source.py:34]: bit of a nit, but possibly just `local_git` or maybe just `local`? The source doesn't have to be a git clone right?

File: libs/datasets/dataset_utils.py [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+    jhu_usa_data = data.get_subset(
+        AggregationLevel.COUNTY, country="USA", after="2020-03-01"
+    ).data
+
+    data = cds_data_source.to_generic_timeseries()
+    cds_usa_data = data.get_subset(
+        AggregationLevel.COUNTY, country="USA", after="2020-03-01"
+    ).data
+
+    # TODO(chris): Better handling of counties that are not consistent.
```
Comment (crwilcox) [inline libs/datasets/dataset_utils.py:74]: worth filing an issue?

File: run.py [hunk 3]
```diff
@@ -1,11 +1,19 @@
...
-import simplejson
+import logging
 import time
 import datetime
+
+import simplejson
 import pandas as pd
-import logging
+
+# from libs.CovidDatasets import CDSDataset, JHUDataset
```
Comment (crwilcox) [inline run.py:8]: remove?

File: libs/datasets/sources/dh_beds.py [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+        for state, county, fips in fips_combos
+    }
+    counties_by_state = defaultdict(list)
+
+    for state, county, _ in fips_combos:
+        counties_by_state[state].append(
+            county.lower()
+            .replace("'", "")
+            .replace("-", " ")
+            .replace("ó", "o")
```
Comment (crwilcox) [inline libs/datasets/sources/dh_beds.py:36]: would it be worth prefetching on some other accented chars? 
Also, is there a reason ó => o but ñ => ""? At least at glance it seems it should also go to a non-accented form?

Comment (ghop02) [reply libs/datasets/sources/dh_beds.py:36]: oh yeah, this was super hacky - I'll look into a better solution for this.  

https://stackoverflow.com/a/2633310

Huh this is cool (example from SO):
```python
accented_string = u'Málaga'
# accented_string is of type 'unicode'
import unidecode
unaccented_string = unidecode.unidecode(accented_string)
# unaccented_string contains 'Malaga'and is of type 'str'
```

**Comment by crwilcox** [comment]
Took another pass, looking good.

File: libs/datasets/sources/dh_beds.py [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+        data[cls.Fields.AGGREGATE_LEVEL] = "county"
+
+        # Backfilling FIPS data based on county names.
+        # TODO: Fix all missing cases
+        fips_data = dataset_utils.build_fips_data_frame()
+        data = match_county_to_fips(data, fips_data)
+        return data
+
+    @classmethod
+    def local(cls) -> "JHUTimeseriesData":
```
Comment (davidstrauss) [inline libs/datasets/sources/dh_beds.py:143]: Why does this beds dataset map to JHUTimeseriesData here?

Comment (ghop02) [reply libs/datasets/sources/dh_beds.py:143]: ah it does not, copy pasta error

**Pull request merged**